types/protocol.d.ts

```diff
@@ -5,6 +5,23 @@
  */
 
 declare module Protocol {
+  /**
+   * An intermediate type, used to create a record of all possible Crdp raw event
+   * messages, keyed on method. e.g. {
+   *   'Domain.method1Name': {method: 'Domain.method1Name', params: EventPayload1},
+   *   'Domain.method2Name': {method: 'Domain.method2Name', params: EventPayload2},
+   * }
+   */
+  type RawEventMessageRecord = {
```

Comment:
this is an unchanged move from below, but it was previously scoped to the file and we need it exported now